src/WireFormat.h

```diff
@@ -659,6 +676,19 @@ struct GetHeadOfLog {
     } __attribute__((packed));
 };
 
+struct GetLeaseInfo {
+    static const Opcode opcode = GET_LEASE_INFO;
+    static const ServiceType service = COORDINATOR_SERVICE;
+    struct Request {
+        RequestCommon common;
+        uint64_t leaseId;
```

Comment:
Document all of the fields in requests and responses (except ridiculously obvious stuff like "common"). Please do this for all of the new request below. There are many requests in WireFormat that have not been properly documented,  but this is a  historical accident.